File: Project 1/Milestone 3/CU Events v1.0/CU Events v1.0/Event.swift
//
//  Event.swift
//  CU Events v1.0
//
//  Created by Mahmoud Aljarrash on 3/4/18.
//  Copyright © 2018 Mahmoud Aljarrash. All rights reserved.
//

import Foundation

struct Event: Decodable
{
    let title: String
    let description: String
    let campus: String
    let address: String
    let startDate: String
    let endDate: String
    
}

File: Project 2/Milestone 3/CUEvents/app/src/main/java/com/jarrash/mahmoud/cuevents/Model/Event.kt
package com.jarrash.mahmoud.cuevents.Model

/**
 * Created by mahmoud on 4/26/18.
 */
class Event
{
    var title: String = ""
    var campus: String = ""
    var address: String = ""
    var startDate: String = ""
    var endDate: String = ""
    var description: String = ""
    var image: String = ""

    constructor() {}

    constructor(title: String, campus: String, address: String,
                startDate: String, endDate: String, description:
                String, image: String)
    {
        this.campus = campus
        this.title = title
        this.startDate = startDate
        this.endDate = endDate
        this.description = description
        this.address = address
        this.image = image


    }
}